src/components/Logout/Logout.jsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import Button from '../shared/Button/Button';
 import { connect } from 'react-redux';
 import { logOut } from '../Login/LoginActions';
-import styles from './Logout.module.css';
 
 export class Logout extends React.Component {
     render() {
         return (
-            <div className={styles.logOut}>
+            <div>
```

Comment:
no need for div

Reply:
Need it for now otherwise the button stretches, planning to make header it's own component soon anyway so will fix it then